pom.xml

```diff
@@ -110,4 +111,11 @@
         <module>cli</module>
     </modules>
 
+    <repositories>
```

Comment:
I'm not sure why, but I couldn't get this build to work again without this remote repo despite this being a `provided` dep in `data-model` ... open to suggestions on how to fix this with local configuration.